libs/pxweb2-ui/src/lib/components/Select/Select.tsx

```diff
@@ -220,7 +225,17 @@ function VariableBoxSelect({
           isOpen={isModalOpen}
           onClose={handleCloseModal}
         >
-          <div className={cl(classes.modalRadioList)}>
+          
```

Comment:
Seems like the indentation is off here, can you format the document?

Reply:
Formatted code as suggested